pkg/async-gateway/main.go

```diff
@@ -20,69 +20,59 @@ import (
 	"flag"
 	"net/http"
 	"os"
-	"strings"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/gorilla/handlers"
 	"github.com/gorilla/mux"
-	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
+
+	"github.com/cortexlabs/cortex/pkg/lib/aws"
+	cr "github.com/cortexlabs/cortex/pkg/lib/configreader"
+	"github.com/cortexlabs/cortex/pkg/lib/errors"
+	"github.com/cortexlabs/cortex/pkg/lib/logging"
+	"github.com/cortexlabs/cortex/pkg/lib/telemetry"
+	"github.com/cortexlabs/cortex/pkg/types/clusterconfig"
+	"github.com/cortexlabs/cortex/pkg/types/userconfig"
 )
 
 const (
 	_defaultPort = "8080"
 )
 
-func createLogger() (*zap.Logger, error) {
-	logLevelEnv := os.Getenv("CORTEX_LOG_LEVEL")
-	disableJSONLogging := os.Getenv("CORTEX_DISABLE_JSON_LOGGING")
-
-	var logLevelZap zapcore.Level
-	switch logLevelEnv {
-	case "DEBUG":
-		logLevelZap = zapcore.DebugLevel
-	case "WARNING":
-		logLevelZap = zapcore.WarnLevel
-	case "ERROR":
-		logLevelZap = zapcore.ErrorLevel
-	default:
-		logLevelZap = zapcore.InfoLevel
+var (
+	gatewayLogger = logging.GetLogger()
+)
+
+func Error(err error, wrapStrs ...string) {
```

Comment:
nit: change name to something more meaningful 

`Error` → `Exit`
or
`Error` → `Fatal`

Reply:
Went with `Exit`. Sounds better this way.